vtex-apps/admin-pages
# Feature - Native types

#### What is the purpose of this pull request?
Implement native types

#### What problem is this solving?
Now if you need the user to input an image to your component, for example, you only need to write in the schema that you want a content of type image and all forms needed will appear in the storefront sidebar. Then, all content inputed by the user will return as a component prop.
Besides the usual types (as image, link, video and text), you can also say that you need a brand and in the storefront sidebar will appear a select component with the names of the available brands, the user will choose one and all brand information will return to you as a component prop. The same reasoning will work for categories, departments and collections. 

#### How should this be manually tested?
You should download and link:
- carousel in the branch feature/new-native-types
- admin in the branch feature/blocks
- store-graphql in the branch feature/brand-category-collection-department-search

Then, open https://{{workspace}}--storecomponents.myvtex.com/admin/cms/storefront?__disableSSR
The __disableSSR is important here.

#### Screenshots or example usage

#### Types of changes
